src/components/ha-selector/ha-selector-area.ts

```diff
@@ -54,7 +54,7 @@ export class HaAreaSelector extends LitElement {
   }
 
   protected willUpdate(changedProperties: PropertyValues): void {
-    if (changedProperties.has("selector") && this.value !== undefined) {
+    if (changedProperties.get("selector") && this.value !== undefined) {
```

Comment:
I guess this could still happen now...

So the idea is to only do it if the selector changed, and not on first updated?